### DISCUSSION TITLE
The plug-in reads concurrent requests from the database

### DISCUSSION THREAD

**Discussion by zzZXMm** [Kong/kong#9223] [Help]
I wrote a plug-in to query the statistical value of the database. In the case of multiple concurrency (about 500), the statistical value is 0 occasionally, but it is not the case when I directly use SQL. I use JMeter to do the test, and there will not be 0 when I directly request SQL concurrently. When 0 is found, the database does not preempt CPU resources. In normal times, it frequently preempts CPU resources. I don't know if there is any cache impact, Has anyone ever encountered this problem?

**Comment by hanshuebner** [comment]
Hello @zzZXMm,

we don't recognize this problem right away. Can you provide a way to reproduce it?

Thank you,
Hans
